Makefile

```
@@ -20,6 +20,7 @@ GRUB := $(shell command -v grub2-mkrescue >/dev/null 2>&1 && echo grub2-mkrescue
 
 all: dep clean installer install-packages squash-root initramfs iso vm
 no-vm: dep clean installer squash-root initramfs iso
+docker: dep clean installer install-packages squash-root docker-image
```

checkmake (0.3.2)

[warning] Target "docker" should be declared PHONY.

(phonydeclared)